CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [21.4] (unreleased)
+## [21.04.0] (unreleased)
```

Comment:
Didn't we want to remove the padding zeroes in the version numbers, especially for Python projects?
Also, you forgot to add the date here.